cmd/telemetrygen/internal/common/config.go

```diff
@@ -62,10 +62,48 @@ func (v *KeyValue) Type() string {
 	return "map[string]any"
 }
 
+// DurationWithInf is a custom type that can handle both regular durations and "inf" value
+type DurationWithInf time.Duration
```

Comment:
Unfortunately, placing this here will break programmatic usage of telemetrygen since it's an internal package. I'd be inclined to put this in `pkg/types` or a similarly named package in order to keep Duration as something which can be set by either the CLI or an external package. 

@atoulme's original idea of setting to continuous if no duration is set made the most sense to me as it's consistent with our `rate` configuration behavior